web/themes/contrib/civictheme/templates/form/input--submit.html.twig

```diff
@@ -17,7 +18,7 @@
 {% set children %}
   {% include "@atoms/button/button.twig" with {
     type: type,
-    text: attributes.value,
```

Comment:
does `attributes.value` contains HTML-converted text? can you try using `raw` filter?